apstra/api_iba_dashboards.go

```diff
@@ -156,7 +170,7 @@ func (o *Client) getIbaDashboardByLabel(ctx context.Context, blueprintId ObjectI
 	return &result[0], nil
 }
 
-func (o *Client) createIbaDashboard(ctx context.Context, blueprintId ObjectId, in *rawIbaDashboard) (ObjectId, error) {
+func (o *Client) createIbaDashboard(ctx context.Context, blueprintId ObjectId, in *IbaDashboardData) (ObjectId, error) {
```

Comment:
Should there be guard rails in the create/marshal process to prevent someone from setting "computed" values (updated by, default) here?

I seem to recall a deadlock situation where a user can create a "predefined" something (dashboard? probe?) and is then prevented from deleting it.

Reply:
Done